libr/bin/mangling/microsoft_demangle.c

```diff
@@ -146,7 +146,7 @@ int copy_string(STypeCodeStr *type_code_str, char *str_for_copy, unsigned int co
 	}
 
 	if (str_for_copy) {
-		strcpy  (dst, str_for_copy);
+		strncpy  (dst, str_for_copy, str_for_copy_len + 1);
```

Comment:
strncpy does not null terminate, better use r_str_ncpy

Reply:
Should be fine by now